app/uk/gov/hmrc/fileupload/s3/S3JavaSdkService.scala

```diff
@@ -211,6 +211,17 @@ class S3JavaSdkService(configuration: com.typesafe.config.Config, metrics: Metri
       "versioningStatus" -> versioningStatus
     )
   }
+
+  def deleteObjectFromBucket(bucketName: String, key: String): Unit = {
+    Try(s3Client.deleteObject(bucketName, key)) match {
+      case Success(_) =>
+        Logger.info(s"Objected successfully deleted with key $key from bucket $bucketName")
+        ()
+      case Failure(error) =>
+        Logger.error(s"Attempted to delete object with key $key from bucket $bucketName but error thrown: ${error.getMessage}")
```

Comment:
There is a tricky thing about our kibana dashboard - only log entries with exceptions are being shown in 'service exceptions' section.
Could we also log the error then ? (i mean adding error as a last parameter of Logger.error())

Reply:
Change pushed.